samples/apps/copilot-chat-app/importdocument/Program.cs

```diff
@@ -120,11 +122,12 @@ private static async Task UploadFileAsync(FileInfo file, Config config, Guid cha
         }
         Console.WriteLine($"Successfully acquired User ID. Continuing...");
 
-        using var fileContent = new StreamContent(file.OpenRead());
-        using var formContent = new MultipartFormDataContent
+        using var formContent = new MultipartFormDataContent();
+        List<StreamContent> filesContent = files.Select(file => new StreamContent(file.OpenRead())).ToList();
```

Comment:
you could get rid of this and just use `foreach (var file in filesContent)` below